app/src/main/java/com/example/marti/unoplus/Card.java

```diff
@@ -1,8 +1,88 @@
 package com.example.marti.unoplus;
 
+import java.io.Serializable;
+
 /**
  * Created by ekzhu on 31.03.2018.
  */
 
-public class Card {
+public class Card implements Serializable{
+
+    //Create 4 different card colors + wild color (for all)
+    public enum colors {
+        RED (0),
+        BLUE (1),
+        GREEN (2),
+        YELLOW (3),
+        WILD (4);
+
+        private int value;
+        private colors(int value){this.value = value;}
+    }
+    public colors color;
+
+    //Create card values
+    public enum values {
+        ZERO (0),
+        ONE (1),
+        TWO (2),
+        THREE (3),
+        FOUR (4),
+        FIVE (5),
+        SIX (6),
+        SEVEN (7),
+        EIGHT (8),
+        NINE (9),
+        SKIP (10),
+        TURN (11),
+        PLUS_TWO (12),
+        CHOOSE_COLOR (13),
+        PLUS_FOUR (14);
+
+        private int value;
+        private values(int value){this.value = value;}
+        }
```

Comment:
Frage könnten wir die Values so implementieren das ich das enum values auch in einer anderen Klasse verwenden kann?

Reply:
Ja, wir können das machen. Es ist schon so. Ich habe diese values auch in Deck Klasse benutzt.

Reply:
ok nice ich bin heute draufgekommen (dank jeni) das es besser ist die Karteneffekte in eine eigene Klasse zu schieben die einfach nur die Values benutzt um den richtigen effekt zu triggern jeder effekt ist eine eigene methode in der klasse und die klasse wird von der Spiellogic verwalted so müssen die Karten nicht direkt auf den GameControler/GameLogic zugreifen weil sogesehen die GameLogic das handelt und die muss nur wissen welche Karte gespielt wurde (das tut sie ja schon) und damit brauchen wir die Cards Klasse nicht irgendwie erweitern oder überfüllen sie mit methoden schließlich wäre dan jeder Karteneffekt in jeder Karte hinterlegt (da wir 100+ Karten haben) was unnötig wäre